tests/cases/optional.cddl

```diff
@@ -0,0 +1,8 @@
+MEM_CONFIG = [
```

Comment:
nit: do we use copyright headers on these types of files?

Reply:
Not right now. I'll look into this more in the future.